app/src/main/java/ch/hikemate/app/model/overpassAPI/OverpassRepository.kt

```diff
@@ -18,8 +19,16 @@ abstract class OverpassRepository {
     /** The type of format to request from the Overpass API, written in OverpassQL. */
     const val JSON_OVERPASS_FORMAT_TAG = "[out:json];"
 
+    /**
+     * The maximum number of hikes that can be requested in a single Overpass API request
+     *
+     * We limit in the [MapScreen], but since some hikes are filtered out, we need to request more
+     * hikes than we actually want to display.
+     */
+    private const val MAX_HIKES_PER_REQUEST = MapScreen.MAX_HIKES_DRAWN_ON_MAP * 3
+
```

Comment:
Question: How did you come up with 3 here ? Do you have any information about what portion of the hike we discard ? Could you do some research ?

Reply:
I just tested on the map, to see when I reach the `MapScreen.MAX_HIKES_DRAWN_ON_MAP` limit, and put a little more. It's purely empirical